www/css/ionic.app.css

```diff
@@ -9882,12 +9882,6 @@ img {
 .list-inset.search-container {
   margin: 15px 10px 5px; }
 
-.side-menu-container {
```

Comment:
It was container for the logo on the side menu, we are not using it anymore. I deleted this class on scss file and it was removed automatically after I rebuild sass.